File: src/main/java/org/minturtle/careersupport/interview/dto/CreateInterviewTemplateResponse.java
package org.minturtle.careersupport.interview.dto;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@Getter
@Builder
@NoArgsConstructor
public class CreateInterviewTemplateResponse {

    private String interviewId;
    private String theme;
}

File: src/main/kotlin/org/minturtle/careersupport/interview/dto/CreateInterviewTemplateResponse.kt
package org.minturtle.careersupport.interview.dto

import org.minturtle.careersupport.interview.entity.InterviewTemplate

data class CreateInterviewTemplateResponse(
    val interviewId: String?,
    val theme: String
){

    companion object{
        fun of(interviewTemplate: InterviewTemplate): CreateInterviewTemplateResponse {
            return CreateInterviewTemplateResponse(
                interviewTemplate.id,
                interviewTemplate.theme
            )
        }
    }
}